cmd/boulder-publisher/main.go

```diff
@@ -73,10 +74,26 @@ func main() {
 		bundle = append(bundle, ct.ASN1Cert(cert.Raw))
 	}
 
+	// TODO(jsha): Publisher is currently configured in production using old-style
```

Comment:
Can you create a follow-up issue to track this? Its easy to lose `TODO` comments.